api.py

```diff
@@ -133,15 +133,19 @@ def wait_for_job(job_id, **kwargs):
     """
     while True:
         time.sleep(0.5)
-        response = make_get_request(job_id, **kwargs)
-        response.raise_for_status()
-
-        if response.ok:
-            msg = response.json()
-            if 'response' in msg:
+        try:
+            response = make_get_request(job_id, **kwargs)
+
+            if response.ok:
```

Comment:
Technically, I think this check is not necessary.  An exception is raised if the response is not OK.